src/MeasureToolbox/integrals.jl

```diff
@@ -596,6 +681,7 @@ end
 # Define default keyword arguments for 1-D integrals
 const UniIntegralDefaults = Dict(:eval_method => Automatic(),
                                  :num_supports => InfiniteOpt.DefaultNumSupports,
+                                 :num_nodes => 3,
                                  :weight_func => InfiniteOpt.default_weight)
```

Comment:
We should initialize this to be empty instead and add a `clear_uni_integral_defaults` method.